bindings/rust/wasmedge-sdk/Cargo.toml

```diff
@@ -15,9 +15,9 @@ anyhow = "1.0"
 num-derive = "0.3"
 num-traits = "0.2"
 thiserror = "1.0.30"
-wasmedge-macro = {path = "../wasmedge-macro", version = "0.3"}
+wasmedge-macro = {workspace = true}
 wasmedge-sys = {path = "../wasmedge-sys", version = "0.13", default-features = false}
```

Comment:
Do we need to make the sys in the workspace just like others.

Reply:
We don't need to do the same thing on `sys` as `macro` and `types` are used by both `sys` and `sdk`. However, `sys` is only used by `sdk`. Thanks!